source/about/mattermost-server-releases.md

```diff
@@ -18,12 +18,12 @@ Mattermost releases a new server version on the 16th of each month in [binary fo
 
 | **Release** | **Released on** | **Support ends** |
 |:---|:---|:---|
-| v9.10 [Download](https://releases.mattermost.com/9.10.0/mattermost-9.10.0-linux-amd64.tar.gz) \| {ref}`Changelog <release-v9-10-feature-release>` | 2024-07-16 | 2024-10-15 |
-| v9.9 [Download](https://releases.mattermost.com/9.9.1/mattermost-9.9.1-linux-amd64.tar.gz) \| {ref}`Changelog <release-v9-9-feature-release>` | 2024-06-14 | 2024-09-15 |
-| v9.8 [Download](https://releases.mattermost.com/9.8.2/mattermost-9.8.2-linux-amd64.tar.gz) \| {ref}`Changelog <release-v9-8-feature-release>` | 2024-05-16 | 2024-08-15 |
+| v9.10 [Download](https://releases.mattermost.com/9.10.1/mattermost-9.10.1-linux-amd64.tar.gz) \| {ref}`Changelog <release-v9-10-feature-release>` | 2024-07-16 | 2024-10-15 |
```

Comment:
If we want to take users directly to the dot release-specific changelog entry, a label is needed for the dot release, and update the changelog links on this page. The label acts as a bookmark to take users to a specific section of the page.

Reply:
We haven't linked to dot release versions before and the dot releases are listed at the top of each changelog entry, so I'm ok with leaving as-is.